src/components/Home.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-
+import dp from '../Images/dp.jpg';
 function Home(props) {
     return (
         <div>
             <h1>Assignment 1</h1>
             <h2>Home page</h2>
```

Comment:
It would be nice if you could replace the <h2> element with an <h1> element and change the text to "Welcome to MIS".